runtime/config.go

```diff
@@ -0,0 +1,102 @@
+package runtime
+
+import (
+	"encoding/json"
+	"os"
+
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	typesPers "github.com/pokt-network/pocket/persistence/types"
+	"github.com/pokt-network/pocket/shared/modules"
+	typesTelemetry "github.com/pokt-network/pocket/telemetry"
+	typesUtil "github.com/pokt-network/pocket/utility/types"
+)
+
+var _ modules.ConsensusConfig = &Config{}
+var _ modules.P2PConfig = &Config{}
+var _ modules.PersistenceConfig = &Config{}
+var _ modules.TelemetryConfig = &Config{}
+var _ modules.UtilityConfig = &Config{}
+
+type Config struct {
```

Comment:
There is a very similar structure in `shared/modules/types.go` that should be consolidated.